src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -4,7 +4,9 @@ import com.hibob.academy.service.PetService
 import jakarta.ws.rs.*
 import jakarta.ws.rs.core.Response
 import jakarta.ws.rs.core.Response.Status
+import org.springframework.stereotype.Controller
 
+@Controller
```

Comment:
remove this, we use jersey annotations for our web layer, not SB

Reply:
fixed, changed to @component instead... now it works as well, thanks!